litmus-portal/frontend/src/components/QuickActionCard/index.tsx

```diff
@@ -28,15 +28,15 @@ const QuickActionCard = () => {
         </Typography>
         <List>
           {/* <QuickActionItems>
-            <img src="icons/cluster.png" alt="cluster" />
+            <img src="/icons/cluster.png" alt="cluster" />
             <Link to="/" className={classes.listItem}>
               Connect a new cluster
             </Link>
           </QuickActionItems> */}
           {userRole === 'Owner' && (
             <QuickActionItems>
               <div className={classes.imgDiv}>
-                <img src="icons/team.png" alt="team" />
```

Comment:
can we use `icons/survey.png` instead of `/icons/survey.png` this creates problems often. same for all icons.

Reply:
@ishangupta-ds  The problem arises when we redirect from one page to another. @S-ayanide suggested me to make these changes for icons